### PR TITLE
New hairstyles 2: Eletric Boogalo

### PR DESCRIPTION
**Pull request by BeeRobynn**
## About the PR
Adds even more new Hairdo's for the crew to flaunt

## Why / Balance
The crew's choices have been getting bland again, so lets give them some more options

## Technical details
Added 11 new hairstyles, sprited once again by Bloomyex, High Court Barber

image
image
image

**Changelog**
:cl:
- add: Added 11 crazy new hairdo's

## Summary by CodeRabbit

- **New Features**
	- Expanded hair accessory options with new markings including various styles such as Hairnet, Long Pompadour, Medium Curls, and more.
	- Enhanced customization for human hair textures with new states for diverse hairstyles, including Frizzy Braid, Queen Bee, and Sagged Mohawk.

- **Bug Fixes**
	- Corrected a typographical error in the "hairnet" entry within the hair states.

### CODE REVIEW

File: Resources/Prototypes/_Goobstation/Entities/Mobs/Customization/Markings/human_hair.yml [hunk 1]
```diff
@@ -102,6 +110,22 @@
   sprites:
     - sprite: _Goobstation/Mobs/Customization/human_hair.rsi
       state: longCurvy
+
```
[the remark this answers is not in the document]
Comment (BeeRobynn) [reply Resources/Prototypes/_Goobstation/Entities/Mobs/Customization/Markings/human_hair.yml]: New Project(39)

File: Resources/Textures/_Goobstation/Mobs/Customization/human_hair.rsi/meta.json [hunk 2]
```diff
@@ -43,6 +43,10 @@
         "name": "frontBraidsShort",
         "directions": 4
       },
+      {
+        "name": "hairnet",
+        "directiions": 4
+      },
```
[the remark this answers is not in the document]
Comment (BeeRobynn) [reply Resources/Textures/_Goobstation/Mobs/Customization/human_hair.rsi/meta.json]: New Project(39)

**Comment by InfinityPandaRed** [comment]
these look really good

**Review by Piras314 — APPROVED** [review]

**Comment by JoeHammad1844** [comment]
I like the pompadour but the star and Mohawk are too much

**Pull request merged**